pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -35,6 +37,11 @@ func init() {
 	defaultMarshaller = regoMarshaller
 }
 
+const capabilitiesNilError = "capabilities object provided by the UVM to the policy engine is nil"
+const invalidError = "Security policy is not valid. Please check security policy or re-generate with tooling."
```

Comment:
nit: don't capitalize error messages.

Reply:
this is technically a message that goes into the JSON so I don't think Go naming conventions apply